### DISCUSSION TITLE
Error C1091 compiler limit: string exceeds 65535 bytes in length (compiling source file ..\..\src\Bullet3OpenCL\NarrowphaseCollision\b3ConvexHullContact.cpp) Bullet3OpenCL_clew satClipHullContacts.h 2110

### DISCUSSION THREAD

**Discussion by e3dos** [bulletphysics/bullet3#3573] [General]
I can't compile. I got this error when trying to build on visual studio.

**Comment by erwincoumans** [comment]
We use Visual Studio every day and compile just fine. What version? How do you create the project solution?

**Comment by e3dos** [comment]
I used build_visual_studio_vr_pybullet_double.bat to build the snl, then opened the sln and compiled the project. It compile most things just fine, but give me this error when compiling bullet3opencl_clew. I'm compiling it in release version x64

**Comment by erwincoumans** [comment]
What version of Visual Studio exactly?

**Comment by e3dos** [comment]
I'm using visual studio community 2015 version 14
